packages/ffe-account-selector-react/less/ffe-account-selector.less

```diff
@@ -5,6 +5,7 @@
 
     position: relative;
     display: block;
+    margin-bottom: 30px;
```

Comment:
Burde man her brukt en av de konstanterna som finnes for avstander?

Reply:
Ja, `@ffe-spacing-lg` er etter alt å dømme riktig variabel. Se [spacing.less](https://github.com/SpareBank1/designsystem/blob/develop/packages/ffe-core/less/spacing.less)